src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/VerifyRegistrationSteps.cs

```diff
@@ -164,7 +164,7 @@ public async Task ThenAEmailDomainErrorIsReturned()
             var errors = JsonConvert.DeserializeObject<List<ErrorItem>>(content);
             errors.Count.Should().Be(1);
             errors[0].PropertyName.Should().BeNull();
-            errors[0].ErrorMessage.Should().Be("Email from verifying user doesn't match registered user");
+            errors[0].ErrorMessage.Should().Be($"Email from verifying user ({_command.Email}) doesn't match registered user ({_registration.Email})");
```

Comment:
Shouldn't do this, GDPR rules mean emails should not be logged (right to be forgotten)